website/community/product-steering/_index.md

```diff
@@ -0,0 +1,57 @@
+
+The committee will schedule a Product Steering meeting, where the proposer presents the proposal. Committee members, interested contributors, and affected sponsors will challenge and discuss the idea. This meeting will be recorded.
+
+Following this, a private go/no-go decision meeting with the affected sponsors will take place. This second meeting will not be recorded, but the outcome will be documented and shared here.
+
+### Why Do We Need It?
+We aim to align sponsored development capacity with Gardener's strategic vision and priorities. Investments should deliver value either to Gardener's users and operators or to the project and its overall reach.
+
+### Who is on The Product Steering Committee?
+The committee is elected by the project's active contributing sponsors and consists of a small number of members (at least three). Currently, the committee members are:
+- [Johannes Scheerer](https://github.com/scheererj) ([Email](mailto:johannes.scheerer@sap.com) and [Slack](https://gardener-cloud.slack.com/archives/D08EXK51QJJ))
+- [Rafael Franzke](https://github.com/rfranzke) ([Email](mailto:rafael.franzke@sap.com) and [Slack](https://gardener-cloud.slack.com/archives/DAQ7R4D6D))
+- [Vedran Lerenc](https://github.com/vlerenc) ([Email](mailto:vedran.lerenc@sap.com) and [Slack](https://gardener-cloud.slack.com/archives/DAQH1NTUL))
+
+### How Does It Work?
+#### 1. Proposal Submission
+- A proposer drafts a **2-5 page product proposal** detailing the idea and shares a link to it with any member of the Product Steering Committee (see contact data above).
+- The proposal is linked on this community page, and a **meeting date** is scheduled **1-4 weeks out** (at least one week for review and preparation but no more than four weeks to ensure timely decisions).
+
+#### 2. Participation
+- Those who wish to **actively challenge or discuss** the proposal can **request an invitation** from any committee member.
+- Others are encouraged to **watch the recording** instead of attending live.
+
+#### 3. Meeting Process
+- The proposer presents the product proposal.
+- The meeting will be **scheduled for 1h**. Presenting the proposal shall take no more than **30 minutes**.
+- Attendees are expected to have **read the proposal in advance and come prepared**.
+- Attendees **provide feedback, ask questions, and challenge the proposal**.
+- The session is **recorded** for transparency.
+
+#### 4. Decision Process
+- A **private meeting** with sponsors follows to make a **go/no-go decision**.
+- The final decision is **documented on this community page**.
+
+### Key Considerations for Topics
+A topic qualifies for Product Steering if it:
```

Comment:
For me, it's not perfectly clear which topics better suite which committee.
What are the key criteria for selecting which committee to propose to?

Reply:
Product: Early state, no design document, just the idea and maybe a prototype. It is about they "Why" question.

Technical: Later state, design document, the decision to do that was already taken. Now it is about the "How" question.

I will add contrasting sentences to both pages.

Reply:
That helps, thanks :)